Team-C/test_wizard/lib/views/teacher_dashboard_view.dart

```diff
@@ -1,88 +1,84 @@
 import 'package:flutter/material.dart';
+import 'package:test_wizard/views/create_base_assessment_view.dart';
+import 'package:test_wizard/views/login_page_view.dart';
+import 'package:test_wizard/views/view_test_view.dart';
```

Comment:
I agree, `view_test_view.dart` is a bit redundant, perhaps just `test_view.dart` or `create_view.dart` as the class within suggests.